legal-api/src/legal_api/resources/v2/business/business_filings/business_filings.py

```diff
@@ -450,7 +450,8 @@ def put_basic_checks(identifier, filing_id, client_request, business) -> Tuple[d
         if not filing_type:
             return ({'message': 'filing/header/name is a required property'}, HTTPStatus.BAD_REQUEST)
 
-        if filing_type not in CoreFiling.NEW_BUSINESS_FILING_TYPES and business is None:
+        if filing_type not in CoreFiling.NEW_BUSINESS_FILING_TYPES and business is None \
+                and filing_type is not CoreFiling.FilingTypes.NOTICEOFWITHDRAWAL:
```

Comment:
add this line for the scenario of a new business (temporary business) trying to withdraw an incorporation application filing